File: src/main/java/io/github/blueminecraftteam/healthmod/client/armor/FaceMaskModel.java
package io.github.blueminecraftteam.healthmod.client.armor;

import net.minecraft.client.renderer.entity.model.BipedModel;
import net.minecraft.client.renderer.model.ModelRenderer;
import net.minecraft.entity.player.PlayerEntity;

// TODO: Model
public class FaceMaskModel extends BipedModel<PlayerEntity> {
    private final ModelRenderer renderer;

    public FaceMaskModel() {
        super(0.0F, 0.0F, 64, 32);

        this.textureHeight = 128;
        this.textureWidth = 128;
        this.renderer = new ModelRenderer(this, 82, 0);
        this.bipedHead.addChild(renderer);
    }
}

File: src/main/kotlin/io/github/blueminecraftteam/healthmod/client/armor/FaceMaskModel.kt
/*
 * Copyright (c) 2020 Blue Minecraft Team.
 *
 * This file is part of HealthMod.
 *
 * HealthMod is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * HealthMod is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License
 * along with HealthMod.  If not, see <https://www.gnu.org/licenses/>.
 */

package io.github.blueminecraftteam.healthmod.client.armor

import net.minecraft.client.renderer.entity.model.BipedModel
import net.minecraft.client.renderer.model.ModelRenderer
import net.minecraft.entity.player.PlayerEntity

// TODO: Model
class FaceMaskModel : BipedModel<PlayerEntity?>(0.0f, 0.0f, 64, 32) {
    private val renderer: ModelRenderer

    init {
        textureHeight = 128
        textureWidth = 128
        renderer = ModelRenderer(this, 82, 0)
        bipedHead.addChild(renderer)
    }
}